mirrord/config/src/feature/env.rs

```diff
@@ -12,6 +12,11 @@ use crate::{
 
 /// Allows the user to set or override the local process' environment variables with the ones
 /// from the remote pod.
+/// Can be set to one of the options:
```

Comment:
```suggestion
///
/// Can be set to one of the options:
```

Extra space for the rendered markdown.